### ISSUE TITLE
[Feature] Expose EmptyBuilder widget when there are no suggestions to show

### ISSUE THREAD

**Issue by Shahmirzali-Huseynov** [maheshmnj/searchfield#21]
I mean when searching "asagds" in the suggestions will write -> no item found. and this text will be custom.
`noItemFoundText :  ,`

https://user-images.githubusercontent.com/49102327/155277988-41ca9e21-b1b4-4457-b652-aa242c1a027f.mov

**Comment by maheshmnj** [comment]
Cool, that's a nice feature request.

**Comment by maheshmnj** [comment]
I  will hopefully get this out by tomorrow.

**Comment by Shahmirzali-Huseynov** [comment]
Great, Thanks.

**Comment by maheshmnj** [comment]
This is now available in `searchfield: ^0.6.2` https://pub.dev/packages/searchfield/changelog

Closing as fixed.

**Comment by maheshmnj** [comment]
<img src="https://github.com/maheshmnj/searchfield/assets/31410839/e70f45c9-b857-4ecc-8cb3-82fc26f3f0e8" width ="300"> 

<details>
<summary>sample code</summary>

```
import 'package:flutter/material.dart';
import 'package:searchfield/searchfield.dart';

class NetworkSample extends StatefulWidget {
  const NetworkSample({Key? key}) : super(key: key);

  @override
  State<NetworkSample> createState() => _NetworkSampleState();
}

class _NetworkSampleState extends State<NetworkSample> {
  final focus = FocusNode();

  Future<List<String>> getSuggestions() async {
    await Future.delayed(const Duration(seconds: 3));
    return [
      'United States',
      'Germany',
      'Washington',
      'Paris',
      'Jakarta',
      'Australia',
      'India',
      'Czech Republic',
      'Lorem Ipsum',
    ];
  }

  var suggestions = <String>[];
  @override
  Widget build(BuildContext context) {
    Widget searchChild(x) => Padding(
          padding: const EdgeInsets.symmetric(vertical: 4.0, horizontal: 12),
          child: Text(x, style: TextStyle(fontSize: 24, color: Colors.white)),
        );
    return Scaffold(
        appBar: AppBar(
          title: Text('Network sample Demo'),
        ),
        floatingActionButton: FloatingActionButton(
          onPressed: () {},
          child: Icon(Icons.add),
        ),
        body: Padding(
          padding: const EdgeInsets.all(8.0),
          child: Column(
            mainAxisAlignment: MainAxisAlignment.center,
            children: [
              SearchField(
                onSearchTextChanged: (query) {
                  final filter = suggestions
                      .where((element) =>
                          element.toLowerCase().contains(query.toLowerCase()))
                      .toList();
                  return filter
                      .map((e) =>
                          SearchFieldListItem<String>(e, child: searchChild(e)))
                      .toList();
                },
                onTap: () async {
                  final result = await getSuggestions();
                  setState(() {
                    suggestions = result;
                  });
                },
                autovalidateMode: AutovalidateMode.onUserInteraction,
                validator: (value) {
                  if (value == null || value.length < 4) {
                    return 'error';
                  }
                  return null;
                },
                emptyWidget: SizedBox(
                    height: 200,
                    child: const Center(child: CircularProgressIndicator())),
                key: const Key('searchfield'),
                hint: 'Search by country name',
                itemHeight: 50,
                scrollbarDecoration: ScrollbarDecoration(),
                onTapOutside: (x) {},
                suggestionStyle:
                    const TextStyle(fontSize: 24, color: Colors.white),
                searchInputDecoration: InputDecoration(
                  focusedBorder: OutlineInputBorder(
                    borderRadius: BorderRadius.circular(24),
                    borderSide: const BorderSide(
                      width: 1,
                      color: Colors.orange,
                      style: BorderStyle.solid,
                    ),
                  ),
                  border: OutlineInputBorder(
                    borderRadius: BorderRadius.circular(24),
                    borderSide: const BorderSide(
                      width: 1,
                      color: Colors.black,
                      style: BorderStyle.solid,
                    ),
                  ),
                  fillColor: Colors.white,
                  filled: true,
                  contentPadding: const EdgeInsets.symmetric(
                    horizontal: 20,
                  ),
                ),
                suggestionsDecoration: SuggestionDecoration(
                  color: Colors.red,
                  border: Border.all(color: Colors.orange),
                  borderRadius: BorderRadius.circular(24),
                ),
                suggestions: suggestions
                    .map((e) =>
                        SearchFieldListItem<String>(e, child: searchChild(e)))
                    .toList(),
                focusNode: focus,
                suggestionState: Suggestion.expand,
                onSuggestionTap: (SearchFieldListItem<String> x) {
                  focus.unfocus();
                },
              ),
            ],
          ),
        ));
  }
}
```
</details>
